src/components/map/Map.tsx

```diff
@@ -220,6 +226,17 @@ const styles = StyleSheet.create({
     flex: 1,
     justifyContent: 'center'
   },
+  logoContainer: {
+    alignItems: 'center',
+    backgroundColor: colors.surface,
+    bottom: 0,
+    height: normalize(30),
+    justifyContent: 'center',
+    left: normalize(13),
+    position: 'absolute',
+    width: normalize(90),
+    zIndex: 1
+  },
```

Comment:
does not work in any case

<img width="363" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/946b0189-4fb8-4609-86e7-76d3b7cca141">

i will update to not break the line

Reply:
<img width="362" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/2eb3592b-e575-48d4-a206-a249de1a22d0">